rubocop.yml

```diff
@@ -1,3 +1,54 @@
 require:
   - rubocop-performance
   - rubocop-rails
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - "bin/*"
+    - "db/**/*"
+    - "lib/**/*"
+    - "log/**/*"
+    - "public/**/*"
+    - "storage/**/*"
+    - "tmp/**/*"
+    - "vendor/**/*"
+    - "Rakefile"
+
+Bundler/DuplicatedGem:
+  Enabled: true
+
+Style/AsciiComments:
+  Enabled: false
+
+Style/SymbolArray:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Style/EmptyMethod:
+  Enabled: true
```

Comment:
Isn't the default value of this option true?

Reply:
Sorry. You are right.
https://docs.rubocop.org/rubocop/cops_style.html#styleemptymethod